ui/src/Components/HomePage.jsx

```diff
@@ -2,19 +2,69 @@ import React from 'react';
 import { Jumbotron, Container } from 'reactstrap';
 import AuthNav from './auth/AuthNav';
 import './App.css';
+import { Card, CardTitle, CardText, CardImg, CardImgOverlay } from 'reactstrap';
+import { Link } from 'react-router-dom';
 
 const HomePage = () => {
     return (
         <React.Fragment>
-            <Jumbotron fluid>
-                <Container fluid>
-                    <h1>Academy of Cleaning</h1>
-                    <p>The academy of cleaning is a platform that advocates for and empowers cleaners to find new business through leveraging the power of social proof!</p>
-                </Container>
-            </Jumbotron>
-            <div className = "authNav">
-                <AuthNav />
+            <Container>
+                <p class="row-buffer">The Academy of Cleaning empowers cleaners like you to find new business.
+                <br />Start your application and get your industry standard cleaning certification today!</p>
+            </Container>
+
+            <Container>
+                <h4 class="row-buffer">Certifications Offered</h4>
+                <div class="row">
+                    <div class="column">
+                        <CardImg  class="image" src="https://media.istockphoto.com/photos/window-cleaning-picture-id162353089?k=6&m=162353089&s=612x612&w=0&h=OcI9R5ilQk_UmgM_kPHg4dCftVIzRsmTIbxJefGPSfM="alt="Windows 101" />
```

Comment:
just to clarify - I assume these images are royalty free, but are you aware if AOC can use these on their site (since more commercial use)?